applications/nrf_desktop/src/modules/usb_state.c

```diff
@@ -37,13 +37,29 @@ LOG_MODULE_REGISTER(MODULE, CONFIG_DESKTOP_USB_STATE_LOG_LEVEL);
 #define CONFIG_USB_HID_PROTOCOL_CODE -1
 #endif
 
+#define USB_HID_SELECTIVE_SUB	\
```

Comment:
we are not paid by line and although we try to keep things short the 80 char limit is generally ignored for longer lines in nrf_desktop

please don't break the line

Reply:
done